app/src/components/pages/collectionsPage/collectionsPage.tsx

```diff
@@ -37,20 +141,106 @@ export const CollectionsPage = ({ data }) => {
         />
         <SearchBar
-          sx={{ maxWidth: "462px", marginTop: "l" }}
+          sx={{ maxWidth: "462px" }}
           id={"search-collections"}
           textInputProps={{
             isClearable: true,
             labelText: "Search by collection title",
-            name: "searchinput",
+            name: "collection_keyword",
             placeholder: "Search by collection title",
+            defaultValue: currentCollectionKeyword,
+            onChange: (e) => handleSearchChange(e),
           }}
-          onSubmit={function (event: React.FormEvent): void {}}
+          onSubmit={handleSearchSubmit}
           labelText={""}
         />
       </Box>
       <HorizontalRule sx={{ marginTop: "xxl", marginBottom: "xxl" }} />
-      <SearchResults showFilter isSearchPage={false} data={data} />
+
+      {isLoaded ? (
+        collections.length > 0 ? (
+          <>
+            <Box sx={{ display: "flex", gap: "xs", marginBottom: "l" }}>
+              <Text
+                sx={{ fontWeight: "500", marginBottom: 0, marginTop: "xs" }}
+              >
+                {" "}
+                Sort by{" "}
+              </Text>{" "}
+              <Menu
+                showSelectionAsLabel
+                showLabel
+                selectedItem={currentSort}
+                labelText={"Sort By"}
+                listItemsData={[
+                  {
+                    id: "date-desc",
+                    label: "Newest to oldest",
+                    onClick: onMenuClick,
+                    type: "action",
+                  },
+                  {
+                    id: "date-asc",
+                    label: "Oldest to newest",
+                    onClick: onMenuClick,
+                    type: "action",
+                  },
+                  {
+                    id: "title-asc",
+                    label: "Title A to Z",
+                    onClick: onMenuClick,
+                    type: "action",
+                  },
+                  {
+                    id: "title-desc",
+                    label: "Title Z to A",
+                    onClick: onMenuClick,
+                    type: "action",
+                  },
+                ]}
+              />
+            </Box>
+            <Heading
+              size="heading5"
+              sx={{ marginBottom: "l" }}
+              ref={headingRef}
+              tabIndex={-1}
+              id={"all-collections-page"}
+              width="max-content"
+            >
+              {`Displaying ${displayResults(
+                data.numResults,
+                data.perPage,
+                data.page
+              )}
+            results`}
+            </Heading>
+            <CardsGrid records={collections} />
+          </>
+        ) : (
+          <NoResultsFound searchTerm={currentCollectionKeyword} />
+        )
+      ) : (
+        Array(Math.ceil(collections.length / 4)).fill(
```

Comment:
Why do you need this here? Note that `CardsGrid` has it's own set of `LaneLoading` so this here bounces around the number of skeleton loaders on the page.

Reply:
In general, and I'm sure I introduced most of these inconsistencies, we should have the loading state at the same level on every page that's displaying cards. I'm not sure which makes more sense (within the CardsGrid/Lane components or the page component) but we should pick one

Reply:
actually, to get the `CardsGrid` loading state to work the way we want it to work, we'll have to pass down the value of "isLoading" as a prop. I think this will require us to update several other paginated pages. 

Due to the existing size of this PR, I don't really want to touch more pages and add MORE features on additional pages to test in the mix. I think we should have a larger conversation about our loading logic and clean this up later.

Reply:
Sounds good.